### PR TITLE
fix: visual alignments 💅

### PR DESCRIPTION
**Pull request by acstll**

### CODE REVIEW

File: packages/components/src/components/progress-bar/progress-bar.css [hunk 1]
```diff
@@ -10,20 +10,25 @@
  */
 
 :host {
-  --background: var(--telekom-color-primary-standard);
-  --background-disabled: var(--telekom-color-ui-disabled);
-  --background-color: var(--telekom-color-ui-faint);
-  --color-error: var(--telekom-color-text-and-icon-functional-danger);
-  --background-error: var(--telekom-color-functional-danger-subtle);
-  --color-success: var(--telekom-color-text-and-icon-functional-success);
+  --track-color: var(--telekom-color-ui-faint);
```
Comment (felix-ico) [inline packages/components/src/components/progress-bar/progress-bar.css:13]: better naming now but will this be a problem/breaking change for people who are using the old variables?

**Review by felix-ico — APPROVED** [review]
i had some questions more than review/feedback. Also checked locally and loos good to me

File: packages/components/src/components/progress-bar/progress-bar.css (hunk 1 above)
Comment (acstll) [reply packages/components/src/components/progress-bar/progress-bar.css:13]: yes, this is "breaking" 🫣 — we can bare a little… 🙏

File: packages/components/src/components/link/link.css [hunk 2]
```diff
@@ -30,54 +30,32 @@
   --line-thickness-visited: var(--line-thickness-initial);
   /* hover */
   --color-hover: var(--telekom-color-text-and-icon-link-hovered);
-  --color-line-hover: var(--telekom-color-functional-interaction-hovered);
-  --line-thickness-hover: var(--line-thickness-initial);
+  --color-line-hover: currentColor;
```
[the remark this answers is not in the document]
Comment (acstll) [reply packages/components/src/components/link/link.css:33]: I remember too, I think this is no longer the case in the latest "spec"

File: packages/components/src/components/data-grid/cell-handlers/number-cell.tsx [hunk 3]
```diff
@@ -39,79 +39,77 @@ export const NumberCell: Cell = {
...
+    }
 
-      return (
-        <p class={`scl-body`} style={{ textAlign: 'right' }}>
-          {value}
-        </p>
-      );
+    // Add prefix/suffix
+    if (prefix || suffix) {
+      value = prefix + value + suffix;
```
[the remark this answers is not in the document]
Comment (acstll) [reply packages/components/src/components/data-grid/cell-handlers/number-cell.tsx:106]: I rather not change that old code haha

File: packages/components/src/components/data-grid/cell-handlers/graph-cell.tsx [hunk 4]
```diff
@@ -33,26 +33,17 @@ export const GraphCell: Cell = {
       )
     );
 
+    // I really don't know the difference between bar and progress
```
[the remark this answers is not in the document]
Comment (acstll) [reply packages/components/src/components/data-grid/cell-handlers/graph-cell.tsx:36]: indeed buggy… this is now "fixed"

**Comment by acstll** [comment]
The relevant visual snapshots pass, only these two in the screenshot randomly failed :shipit: 

image

**Pull request merged**